docs/semgrep-supply-chain/sbom.md

```diff
@@ -23,7 +23,7 @@ Semgrep Supply Chain supports the following:
 
 :::info Prerequisites
 - **SBOM generation** can be performed only through Semgrep AppSec Platform.
-- You need at least one successful Supply Chain scan on the **trunk** branch of each repository you want to generate an SBOM for. See [<i class="fa-regular fa-file-lines"></i> Core deployment](/deployment/core-deployment) to set up your Semgrep account and Supply Chain scans.
+- The latest successful full scan running Supply Chain must have run on the **trunk** (default) branch of each repository you want to generate an SBOM for, or there can be a mismatch between dependencies and vulnerabilities. See [<i class="fa-regular fa-file-lines"></i> Core deployment](/deployment/core-deployment) to set up your Semgrep account and Supply Chain scans.
```

Comment:
Not sure if I got the details correct, but I wanted to reword a bit since the first sentence has a few criteria that must be met; I'm also not sure what the specifics of the mismatch would be. WDYT of something like this?

> You need at least one successful Supply Chain scan on the **trunk** (default) branch of each repository you want to generate an SBOM for, and the most recent scan must be a full scan. Otherwise, there can be issues associating the vulnerabilities with the correct dependencies.

Reply:
No, that's not quite right unfortunately. The most recent full scan for the repo must be on the default branch. Otherwise, when we create the SBOM, the vulnerabilities are drawn from the default branch, but the dependencies will be drawn from the latest full scan, whatever branch that is. So it has to be the default for the two to match.

Reply:
if we need to pull this out of the callout bc it's more complex, totally open to that!

Reply:
Yeah, I think it would be helpful to pull it out into its own section and then list the reqs out with bullet points. WDYT?

Reply:
I'll give it a shot!